anitya/db/models.py

```diff
@@ -896,16 +898,21 @@ class User(Base):
     email = sa.Column(sa.String(256), nullable=False, index=True, unique=True)
     username = sa.Column(sa.String(256), nullable=False, index=True, unique=True)
     active = sa.Column(sa.Boolean, default=True)
+    admin = sa.Column(sa.Boolean, default=False)
 
     @property
-    def admin(self):
+    def is_admin(self):
         """
-        Determine if this user is an administrator.
+        Determine if this user is an administrator. Set admin flag
+        if the user is preconfigured.
 
         Returns:
             bool: True if the user is an administrator.
         """
-        return six.text_type(self.id) in anitya_config.get('ANITYA_WEB_ADMINS', [])
+        if not self.admin:
+            if six.text_type(self.id) in anitya_config.get('ANITYA_WEB_ADMINS', []):
```

Comment:
I think that it would be good to drop support for this config key. That way there would be only one place where you can manage admins.

Reply:
The admins set in the configuration file shouldn't be stripped of this privilege. This is failsafe against some admin going rampage and revoking rights for everyone else and yourself. Also the configuration file could be changed in ansible, so you can easily remove or add admin even if you don't have the admin rights in the applications itself.

Reply:
Ok, I was just concerned that with time it will  not be obvious where the admins are managed (ie config or in the db).